source/common/ssl/context_impl.cc

```diff
@@ -342,7 +342,7 @@ X509Ptr ContextImpl::loadCert(const std::string& cert_file) {
   if (!fp.get() || !PEM_read_X509(fp.get(), &cert, nullptr, nullptr)) {
     throw EnvoyException(fmt::format("Failed to load certificate '{}'", cert_file.c_str()));
   }
-  return X509Ptr{cert};
+  return X509Ptr(cert);
```

Comment:
This is most likely wrong change, but it everywhere else in the code `SslConPtr`, `SslCtxPtr` and `X509Ptr` are using `()` and not `{}` for initialization / casting.